src/sentry/notifications/notifications/rules.py

```diff
@@ -222,9 +222,11 @@ def get_context(self) -> MutableMapping[str, Any]:
         environment = self.event.get_tag("environment")
         enhanced_privacy = self.organization.flags.enhanced_privacy
         rule_details = get_rules(self.rules, self.organization, self.project)
+        group = self.group
+        group.users_seen = self.group.count_users_seen()
```

Comment:
Could this be added as context data instead of mutating the group? Is `users_seen` gonna be used further downstream?

Reply:
I can add it as a part of the context.